db/migrate/20190722114341_add_proxy_domains.rb

```diff
@@ -0,0 +1,20 @@
+class AddProxyDomains < ActiveRecord::Migration
+  def change
+    add_column :proxies, :staging_domain, :string
+    add_column :proxies, :production_domain, :string
+
+    add_index :proxies, [ :staging_domain, :production_domain ]
```

Comment:
The migrations are not done for `postgres` and `oracle` schemas. I tried to do it locally and saw this error (this is a blocking migration):
![image](https://user-images.githubusercontent.com/11318903/61859164-e11cd980-aec7-11e9-9038-b6100f115e19.png)

Thankfully it suggests a way:
```ruby
add_index :proxies, [:staging_domain, :production_domain], algorithm: :concurrently
```

Reply:
But that works for mysql too? Honestly I have no idea how you handle those.

Reply:
buff, what it suggests is not that easy:
![image](https://user-images.githubusercontent.com/11318903/61859319-33f69100-aec8-11e9-899c-b8c57c31d633.png)

![image](https://user-images.githubusercontent.com/11318903/61859341-3e188f80-aec8-11e9-9394-cd876ceaf7aa.png)

I guess it should be done in more than 1 migration and the add index should be a migration of its own 🤔

Reply:
> But that works for mysql too?

No idea, we gotta try 🤷‍♀ I am trying one that works for all yet 😂

Reply:
![image](https://user-images.githubusercontent.com/11318903/61859708-ee869380-aec8-11e9-991e-4c908c67c7bb.png)

![image](https://user-images.githubusercontent.com/11318903/61859729-f7776500-aec8-11e9-9140-a72fe2f92c40.png)

So I will add a condition for the kind of DB 🤔